src/python/espressomd/analyze.pyx

```diff
@@ -192,13 +192,13 @@ class Analysis(object):
     # Analyze center of mass
     #
 
-    def center_of_mass(self, part_type=None):
+    def center_of_mass(self, p_type=None):
```

Comment:
Why not make it a plain argument, then nobody has to remember the name of the keyword and we don't have to check if it was provided or not.

Reply:
There definitely needs to be some kind of cleanup regarding this. Sometimes it is "p_type", "part_type" or just "type". Sometimes it has to be a single int, othertimes a list of ints and sometimes "all". It would be great if the all the analysis routines could just take a ParticleSlice for exampe from particle_data.ParticleHandle.select()